jekyll/features.html

```diff
@@ -9,19 +9,32 @@ <h1 class="text-centered page-title main-heading">Features</h1>
   <section class="content">
     <div class="pure-g">
       <div class="pure-u-1">
-        <h1>Intuitive and clean syntax</h1>
+        <h1>One language to rule them all</h1>
       </div>
       <div class="pure-u-1 pure-u-md-1-2 code-align">
-        <h2>
-          Taking inspiration from multiple languages, the Nim syntax is
-          easily understood and results in code that can be modified very
-          effectively.
-        </h2>
+        <h2>Good for everything, from shell scripting to web front and backend,
+          to machine learning, high-performance computing (HPC), and embedded.</h2>
         <ul>
-          <li>Indentation significant syntax</li>
-          <li>Multiple constructs inspired by Python</li>
-          <li>Multi-line lambdas</li>
-          <li>It combines successful concepts from Ada and Modula.</li>
+          <li>Productivity: human-readable and expressive like Python</li>
+          <li>Type system: strong static typing, nice generics, concepts, and type classes</li>
+          <li>Performance: matches C and C++, always gives control to the programmer</li>
+          <li>Portability: compiles to C, C++, and Javascript
```

Comment:
Mention something like `(LLVM also available)` ?. 🤔

Reply:
```suggestion
          <li>Portability: compiles to C, C++, and Javascript/NodeJS (LLVM also available)
```


If someone can not compile to Node, [open bug here](https://github.com/juancarlospaco/nodejs/issues)

Reply:
But NodeJS *is* JavaScript, no? We're talking about languages, not runtimes here

Reply:
Yes, and we also shouldn't be exhaustive. We should focus on what we support officially. LLVM and NodeJS isn't it.